apps-rendering/src/components/Layout/NewsletterSignUpLayout.tsx

```diff
@@ -60,9 +104,38 @@ const NewsletterSignUpLayout: FC<Props> = ({ item, children }) => {
 					/>
 				</header>
 				<section css={contentRow}>
+					{!!newsletter && (
+						<div css={detailBlockStyles}>
+							<SvgNewsletter size="xsmall" />
+							<b>{newsletter.frequency}</b>
+							{/* TO DO - use regional focus, when on the MAPI type */}
+						</div>
+					)}
 					<Headline item={item} />
 					<Standfirst item={item} />
-					<Body format={item}>{children}</Body>
+
+					{!!newsletter && (
+						<div css={frequencyBlockStyles}>
+							<SvgClock size="xsmall" />
+
+							<span>
+								You&apos;ll receive this newsletter
+								<b> {newsletter.frequency}</b>
+							</span>
+						</div>
+					)}
+
+					{!!newsletter && (
+						<InPageNewsletterSignup
+							newsletter={newsletter}
+							format={getFormat(item)}
+							fallbackContent={
+								<ArticleBody format={item}>
+									{children}
+								</ArticleBody>
+							}
+						/>
+					)}
```

Comment:
we could use `maybeRender` here too

```suggestion
					{maybeRender(item.promotedNewsletter, (newsletter) => (
						<>
							<div css={frequencyBlockStyles}>
								<SvgClock size="xsmall" />

								<span>
									You&apos;ll receive this newsletter
									<b> {newsletter.frequency}</b>
								</span>
							</div>

							<InPageNewsletterSignup
								newsletter={newsletter}
								format={getFormat(item)}
								fallbackContent={
									<ArticleBody format={item}>
										{children}
									</ArticleBody>
								}
							/>
						</>
					))}
```